pkg/apis/options/options.go

```diff
@@ -243,6 +245,7 @@ func NewFlagSet() *pflag.FlagSet {
 	flagSet.Bool("ssl-insecure-skip-verify", false, "skip validation of certificates presented when using HTTPS providers")
 	flagSet.Bool("ssl-upstream-insecure-skip-verify", false, "skip validation of certificates presented when using HTTPS upstreams")
 	flagSet.Duration("flush-interval", time.Duration(1)*time.Second, "period between response flushing when streaming responses")
+	flagSet.Duration("proxy-time-out", time.Duration(30)*time.Second, "timeout for upstream proxy")
```

Comment:
Could you expand on the description of this flag a little, something along the lines of `timeout for requests to upstream servers, if response is not returned within timeout, something happens`

Reply:
Update name and description